request-processor/src/tasks.py

```
@@ -141,6 +143,103 @@ def handle_check_file(request_schema, request_data, tmp_dir):
+                {"endpoint": endpoint, "pipelines": dataset}
+            )
+        ]
+        documentation_urls = [
+            source.get("documentation-url")
+            for source in source_rows
+            if source.get("documentation-url")
+        ]
+        licences = [
+            source.get("licence") for source in source_rows if source.get("licence")
+        ]
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)